src/transactions/feed/queryHelper.ts

```diff
@@ -78,9 +78,7 @@ export const deduplicateTransactions = (
     transactionsByTxHash[transaction.transactionHash] = transaction
   })
 
-  return Object.values(transactionsByTxHash).sort((a, b) => {
```

Comment:
i wonder if this is safe to remove?

Reply:
yes i moved it to the TransactionFeed - we don't actually need to sort the transactions in the query helper, we only need to sort it for displaying the `confirmedFeedTransactions`